### PR TITLE
403 error screen

### PR DESCRIPTION
**Pull request by sambailey-moj**

### CODE REVIEW

File: integration_tests/integration/forbidden.spec.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    cy.task('stubGetUnallocatedCase')
+    cy.visit('/pdu/PDU1/J678910/convictions/1/case-view', { failOnStatusCode: false })
+  })
+
+  it('403 message should be shown', () => {
+    const forbiddenPage = Page.verifyOnPage(ForbiddenPage)
+    forbiddenPage.message().should('exist')
+  })
+
+  it('403 header/title should be shown', () => {
```
Comment (GaReid) [inline integration_tests/integration/forbidden.spec.ts:18]: Do we really need two tests for this. Couldn't we check the message and the heading in the same test '403 page should be shown'

**Review by GaReid — APPROVED** [review]
Added a suggestion re tests ,but LGTM

File: integration_tests/integration/forbidden.spec.ts (hunk 1 above)
Comment (sambailey-moj) [reply integration_tests/integration/forbidden.spec.ts:18]: fixed!

Comment (GaReid) [reply integration_tests/integration/forbidden.spec.ts:18]: Have you pushed the change?

**Review by GaReid — APPROVED** [review]

**Pull request merged**